packages/admin/cms-admin/src/index.ts

```diff
@@ -20,6 +20,8 @@ export { Publisher } from "./builds/Publisher";
 export { includeInvisibleContentContext } from "./common/apollo/links/includeInvisibleContentContext";
 export { DropdownMenuItem } from "./common/DropdownMenuItem";
 export { AuthorizationErrorPage } from "./common/errors/AuthorizationErrorPage";
+export { BuildInformationContext } from "./common/header/about/build-information/BuildInformationContext";
```

Comment:
We should not export the context and a provider. Maybe we could also add a `useBuildInformation` hook to completely hide the "private" BuildInformationContext"?

Reply:
I decided not to create "useBuildInformation", as I didn't want to add our usual `undefined` check, because the provider should be optional.

```
export function useBuildInformation(): BuildInformation {
    return React.useContext(BuildInformationContext);
}
```
should I add this?

Reply:
We usually have a pair of `<Name>Provider` and `use<Name>` to hide the context itself. There's no need to add an undefined check if the context may not be defined.

> should I add this?

I'd say yes!

Reply:
done